thesis-api-v2/src/main/resources/search_phrases.txt

```diff
@@ -0,0 +1,11 @@
+name;search_word;quantity
+animals;types of mammals;100
+animals;types of lizards;100
+animals;types of birds;100
+cars; types of cars;100
```

Comment:
Is it intended to have a white character before `search_word`? Sure, it makes so that we can have two semi identical `search_word` for two different categories, but it is confusing and I would like to get rid of this approach. If it is just a typo then... please just remove it.

Reply:
there should not be white character and not two semi identical search_words either, I have to spend more time to make a list of all wanted categories and search phrases, this files will change